### PR TITLE
feat(plugin-google-analytics): add debug option

### PR DESCRIPTION
**Pull request by EdieLemoine**
Adds the ability to enable DebugView in development.

More information on DebugView: https://support.google.com/analytics/answer/7201382

### CODE REVIEW

**Review by Mister-Hope — APPROVED** [review]

**Comment by EdieLemoine** [comment]
What's left for this PR to be merged? I see the coverage has decreased, but this plugin wasn't covered at all before this change.

**Comment by meteorlxy** [comment]
Sorry for delay. Didn't have time to look into this. I'll handle it after #1030 (a huge refactor :man_facepalming:)

File: packages/@vuepress/plugin-google-analytics/src/client/composables/useGoogleAnalytics.ts [hunk 1]
```diff
@@ -41,5 +45,10 @@ export const useGoogleAnalytics = (id: string): void => {
   }
 
   gtag('js', new Date())
-  gtag('config', id)
+
+  if (options.debug) {
+    gtag('config', options.id, { debug_mode: true })
+  } else {
+    gtag('config', options.id)
+  }
```
Comment (meteorlxy) [inline packages/@vuepress/plugin-google-analytics/src/client/composables/useGoogleAnalytics.ts:53]: Can we use `gtag('config', options.id, { debug_mode: options.debug === true })` ? I'm not sure if setting `{ debug_mode: false }` will cause any different behavior

File: docs/reference/plugin/google-analytics.md [hunk 2]
```diff
@@ -55,3 +55,24 @@ module.exports = {
...
 ```
+
+### debug
+
+- Type: `boolean`
+
+- Details:
+
+  Set to true to enable sending events to DebugView in development.
+
```
Comment (meteorlxy) [inline docs/reference/plugin/google-analytics.md:66]: It would be nice to add a `- Also see: ` and add the link to the debug mode docs.

**Review by meteorlxy — CHANGES_REQUESTED** [review]
Could you help to resolve the conflicts with main branch?

File: packages/@vuepress/plugin-google-analytics/src/client/composables/useGoogleAnalytics.ts (hunk 1 above)
Comment (EdieLemoine) [reply packages/@vuepress/plugin-google-analytics/src/client/composables/useGoogleAnalytics.ts:53]: That would make sense and look nice, but unfortunately they said this on the page I linked in the description:

> To disable debug mode, exclude the 'debug_mode' parameter; setting the parameter to 
 false doesn't disable debug mode.

**Review by meteorlxy — APPROVED** [review]

**Pull request merged**